Company-Preperation/Goldman-Sachs-2020/GoldmanSachs1.py

```diff
@@ -0,0 +1,26 @@
+'''

```

Comment:
Great work! Do keep pushing more for company hiring processes you've gone through! 💯